app/assets/javascripts/area_served.js

```diff
@@ -0,0 +1,45 @@
+
+
+
+function calculate_client_share_total(){
```

Comment:
The "new" way of doing things is using Stimulus controllers. You can look in `app/javascript` for some examples.

Reply:
Yeah.   I've taken a look at stimulus, and it's not obvious to me how to convert this.   Might be a thing to pair on, but I'm inclined to let the thing that is working work - maybe put in a technical debt item for someone to work on converting both this and the distribution edits, which will be similar.

Reply:
It has been converted.